integration-tests/integration_tests/integration_tests/end_to_end_tests/int_asynchronous_express_messaging_pattern_tests.py

```diff
@@ -64,17 +64,18 @@ def test_should_return_successful_response_from_spine_to_message_queue(self):
         self.assertions.hl7_xml_contains_response_code_and_patient_id(hl7_xml_message_assertor)
         self.assertions.message_status_recorded_as_successfully_processed(state_table_assertor, message_id)
 
-    def test_should_return_successful_response_and_record_spine_reply_in_resync_table_if_sync_async_requested(self):
+    def test_should_return_successful_response_and_record_spine_reply_in_resync_table_if_wait_for_response_requested(self):
         # Arrange
         messages = [build_message('QUPC_IN160101UK05', '9689177923') for _ in range(1)]
 
         # Act
-        responses = send_messages_concurrently(messages, interaction_id='QUPC_IN160101UK05', sync_async=True)
+        responses = send_messages_concurrently(messages, interaction_id='QUPC_IN160101UK05', wait_for_response=True)
 
         # Assert
         all_sync_async_states = MHS_SYNC_ASYNC_TABLE_DYNAMO_WRAPPER.get_all_records_in_table()
         if has_errors(responses):
-            # TODO: NIAD-72 remove prints and has_errors once sync-async feature is stable
+            # TODO: NIAD-72 remove prints and has_errors once wait-for-response feature is stable
+            # TODO: 2020-05-14 should it be removed now? please comment while review
```

Comment:
I think it should be removed as we don't have pending bugs for it

Reply:
I'll delete it :)